sys/linux/dev_img_rogue.txt

```diff
@@ -0,0 +1,1925 @@
+	ui32ReadLen	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLRELEASEDATA {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLRESERVESTREAM {
+	hSD		IMG_HANDLE
+	ui32Size	int32
+	ui32SizeMin	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLRESERVESTREAM {
+	ui32BufferOffset	int32
+	ui32Available		int32
+	eError			int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLWRITEDATA {
+	hSD		IMG_HANDLE
+	ui32Size	len[psData, int32]
+	psData		ptr[in, array[int8]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLWRITEDATA {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCLEARBREAKPOINT {
+	hPrivData	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCLEARBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXDISABLEBREAKPOINT {
+	hPrivData	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXDISABLEBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXENABLEBREAKPOINT {
+	hPrivData	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXENABLEBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXOVERALLOCATEBPREGISTERS {
+	ui32TempRegs	int32
+	ui32SharedRegs	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXOVERALLOCATEBPREGISTERS {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETBREAKPOINT {
+	hPrivData		IMG_HANDLE
+	eFWDataMaster		int32
+	ui32BreakpointAddr	int32
+	ui32HandlerAddr		int32
+	ui32DM			int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCREATECOMPUTECONTEXT {
+	ui32Priority				int32
+	ui32FrameworkCmdize			len[psFrameworkCmd, int32]
+	psFrameworkCmd				ptr[in, array[int8]]
+	hPrivData				IMG_HANDLE
+	ui32StaticComputeContextStateSize	len[psStaticComputeContextState, int32]
+	psStaticComputeContextState		ptr[in, array[int8]]
+	ui32PackedCCBSizeU88			int32
+	ui32ContextFlags			int32
+	ui64RobustnessAddress			int64
+	ui32MaxDeadlineMS			int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCREATECOMPUTECONTEXT {
+	hComputeContext	IMG_HANDLE
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXDESTROYCOMPUTECONTEXT {
+	hComputeContext	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXDESTROYCOMPUTECONTEXT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXFLUSHCOMPUTEDATA {
+	hComputeContext	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXFLUSHCOMPUTEDATA {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXGETLASTCOMPUTECONTEXTRESETREASON {
+	hComputeContext	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXGETLASTCOMPUTECONTEXTRESETREASON {
+	ui32LastResetReason	int32
+	ui32LastResetJobRef	int32
+	eError			int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXKICKCDM2 {
+	hComputeContext			IMG_HANDLE
+	ui32ClientCacheOpSeqNum		int32
+	ui32ClientUpdateCount		len[phClientUpdateUFOSyncPrimBlock, int32]
+	phClientUpdateUFOSyncPrimBlock	ptr[in, array[IMG_HANDLE]]
+	pui32ClientUpdateOffset		ptr[in, array[int32]]
+	pui32ClientUpdateValue		ptr[in, array[int32]]
+	hCheckFenceFd			PVRSRV_FENCE
+	hUpdateTimeline			PVRSRV_TIMELINE
+	puiUpdateFenceName		ptr[in, array[int8]]
+	ui32CmdSize			len[psDMCmd, int32]
+	psDMCmd				ptr[in, array[int8]]
+	ui32PDumpFlags			int32
```

Comment:
Same as above. There seem to be some other occurrences of ui32PDumpFlags as well.

Reply:
Fixed.